src/language-server/Contextive.LanguageServer/Server.fs

```diff
@@ -22,6 +28,24 @@ let version =
         .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
         .InformationalVersion
 
+let private getWorkspaceFolder (s: ILanguageServer) =
+    let workspaceFolders = s.WorkspaceFolderManager.CurrentWorkspaceFolders
+
+    if not (Seq.isEmpty workspaceFolders) then
+        let workspaceRoot = workspaceFolders |> Seq.head
+        Some <| workspaceRoot.Uri.ToUri().LocalPath
+    else
+        None
+
+let private getContextiveDefinitionsPath (workspaceFolder: string) =
+    let combine b a = Path.Combine(a, b)
+
+    let contextiveFolder = workspaceFolder |> combine contextiveFolderName
+
+    if Directory.Exists contextiveFolder 
```

Comment:
Can remove this check - constructing the default setting value doesn't need to check it's validity.  Other parts of the codebase already have to handle the scenario that a configured location doesn't exist.

Reply:
In fact, I'm not sure we need to combine with the workspaceFolder at all.

The method below, (`getConfig`) usually returns a relative path from the setting value supplied by VsCode.  So the new `getDefaultContextiveDefinitionsPath` can just return the relative path as well.  Combining with the workspace folder can then happen later in the same way for both cases.

Reply:
By the way sorry for rebasing this already, the comments are no longer pointing to the right place and in hindsight I should have used fixup commits instead and rebase as the last operation.

I changed this to just be a literal value `let defaultContextiveDefinitionsPath = ".contextive/definitions.yml"`